R/get_categorical_field_labels.R

```diff
@@ -13,6 +13,10 @@ get_categorical_field_labels <- function(metadata) {
     dplyr::filter(field_type %in% c("checkbox", "radio", "yesno", "dropdown")) %>%
     dplyr::select(c(field_name, field_type, field_label, form_name, select_choices_or_calculations))
 
+  # Fill out categorical data for fields of type 'yesno'
+  categorical_fields$select_choices_or_calculations[categorical_fields$select_choices_or_calculations == "" &
```

Comment:
This will throw an error if the metadata has a factor for select_choices_or_calculations as it was in the test dataset used in the testthat tests. I feel like it's the test dataset that is wrong. I'll provide an update to the test data set and the script that builds it.